server/tests/taskratings.test.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from '../../shared/types/customTypes';
 import { withoutPermission } from './testUtils';
 import TaskRating from '../src/api/taskratings/taskratings.model';
+import { convertScale, revertScale } from '../../shared/utils/conversion';
```

Comment:
There are still quite a few raw, unconverted values. Each one should be converted first, even the `-1` would be good to convert